recipes/orange-spectroscopy/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "orange-spectroscopy" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Orange-Spectroscopy-{{ version }}.tar.gz
+  sha256: 7aa505796603ad580bb0290e49e56233c193726b912c18905cd847abb5d2849f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - anyqt >=0.0.6
+    - bottleneck
+    - colorcet
+    - extranormal3
```

Comment:
This will cause a race condition when we merge and you'll have to restart the `orange-spectroscopy` after an hour or so in order to find `extranormal3` in the channel.